src/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,245 @@
+import * as classNames from "classnames";
+import * as React from "react";
+import { useState, useRef } from "react";
+import { useCombobox, useMultipleSelection } from "downshift";
+import * as FontAwesome from "react-fontawesome";
+
+import Label from "../Label/Label";
+import { FormElementSize, formElementSizeClassName } from "../utils/Forms";
+import { Values } from "../utils/types";
+
+import "./MultiSelect.less";
+
+// value must be provided for searchability
+type Option = { key: string; value: string; renderedComponent?: React.ReactNode };
```

Comment:
It feels like we might be able to get away with two properties here instead of three. What if we dropped `key`, and used value for both purposes? `value` is what the HTML `<select>` uses as its identifier. 

How do you feel about shortening `renderedComponent` to `content`?

Reply:
yeah sounds good